truss/templates/server/model_wrapper.py

```diff
@@ -30,11 +27,13 @@ class Status(Enum):
     _status: Status = Status.NOT_READY
     _logger: logging.Logger
     ready: bool
+    name: str
 
     def __init__(self, config: Dict):
-        super().__init__(MODEL_BASENAME)
         self._config = config
         self.logger = logging.getLogger(__name__)
```

Comment:
I think this is an exsting bug here. These logger calls should be `self._logger` instead of `self.logger`